pkg/auditlog/README.md

```diff
@@ -0,0 +1,35 @@
+# alog
+
+A Go package to asynchronously send audit logs events to WorkOS.
+
+## Install
+
+```sh
+go get -u github.com/workos-inc/workos-go/pkg/auditlog
+```
+
+## How it works
+
+```go
+package main
+
+import "github.com/workos-inc/workos-go/pkg/auditlog"
+
+func main() {
+    alog.SetAPIKey("my_api_key")
```

Comment:
Why the shortname? Does `auditlog.*` work?

Reply:
Had a similar thought, I would expect the package name to match the last path of the import.

Reply:
For verbosity, using a package in go requires that each functions or structs are prefixed by `pkgname.`. For this reason we usually try to keep package name short.

I'm not really opinionated about this so if we really feels it an important tradeoff in readability, just let it rename it `auditlog`.

Reply:
Ah, good point. But I think we should also value continuity between the Go and Node sdk. Would you mind reverting it to `auditlog` to preserve this?

Reply:
From the [go docs](https://blog.golang.org/package-names):

> By convention, the last element of the package path is the package name

Reply:
reverted